commit 613f4b3ed7902d1dbbc6ade6401e452a63dfbc21
Author: Kees Cook <kees@kernel.org>
Date:   Tue May 3 13:55:01 2022 -0700

    randstruct: Split randstruct Makefile and CFLAGS
    
    To enable the new Clang randstruct implementation[1], move
    randstruct into its own Makefile and split the CFLAGS from
    GCC_PLUGINS_CFLAGS into RANDSTRUCT_CFLAGS.
    
    [1] https://reviews.llvm.org/D121556
    
    Cc: linux-hardening@vger.kernel.org
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Link: https://lore.kernel.org/r/20220503205503.3054173-5-keescook@chromium.org

diff --git a/Makefile b/Makefile
index 29e273d3f8cc..91c91fcf3c24 100644
--- a/Makefile
+++ b/Makefile
@@ -1011,6 +1011,7 @@ include-$(CONFIG_KASAN)		+= scripts/Makefile.kasan
 include-$(CONFIG_KCSAN)		+= scripts/Makefile.kcsan
 include-$(CONFIG_UBSAN)		+= scripts/Makefile.ubsan
 include-$(CONFIG_KCOV)		+= scripts/Makefile.kcov
+include-$(CONFIG_RANDSTRUCT)	+= scripts/Makefile.randstruct
 include-$(CONFIG_GCC_PLUGINS)	+= scripts/Makefile.gcc-plugins
 
 include $(addprefix $(srctree)/, $(include-y))